### ISSUE TITLE
Inkscape crashes when launching textext

### ISSUE THREAD

**Issue by mattfack** [textext/textext#341]
Hi
I am opening this as blank since it might be a personal situation.

I have installed inkscape 1.1.1 on windows 10 and I installed succesfully also textext 1.7.1 (see the log).

However, whenever I start textext Inkscape freezes than crashes and I can't understand why. The only thing I can think of is that there are some issues with the installation done with admin rights and the installation of textext done in the user folder. Anyway, I can see the extension, but it does not start. Even the extension manager freezes inkscape all together.

Any help would be much appreciated

[TexText][WARNING ]: Bad "inkscape" executable: "C:\Users\username\inkscape\inkscape-1.1.1_2021-09-20_3bf5ae0d25-x64\idir\bin\inkscape.exe"
[TexText][WARNING ]: Fall back to automatic detection of "inkscape"
[TexText][INFO    ]: Using "pdflatex-executable" = "C:\Users\username\AppData\Local\Programs\MiKTeX\miktex\bin\x64\pdflatex.exe"
[TexText][SUCCESS ]: + [Succ] TexText requirements
[TexText][SUCCESS ]: /-and-* [Succ] Detect inkscape>=1.0
[TexText][SUCCESS ]: |      inkscape=1.1 is found
[TexText][SUCCESS ]: /-and-+ [Succ] Detect *latex
[TexText][SUCCESS ]: |     /--or-* [Succ] pdflatex is found at "C:\Users\username\AppData\Local\Programs\MiKTeX\miktex\bin\x64\pdflatex.exe"
[TexText][SUCCESS ]: |     /--or-* [Succ] "lualatex.exe" is found at "C:\Users\username\AppData\Local\Programs\MiKTeX\miktex\bin\x64"
[TexText][SUCCESS ]: |     |      "lualatex.exe" is found at "C:\Users\username\AppData\Local\Programs\MiKTeX\miktex\bin\x64\"
[TexText][SUCCESS ]: |     /--or-* [Succ] "xelatex.exe" is found at "C:\Users\username\AppData\Local\Programs\MiKTeX\miktex\bin\x64"
[TexText][SUCCESS ]: |            "xelatex.exe" is found at "C:\Users\username\AppData\Local\Programs\MiKTeX\miktex\bin\x64\"
[TexText][SUCCESS ]: /-and-+ [Succ] Detect GUI library
[TexText][SUCCESS ]:       /--or-* [Succ] GTK3 is found
[TexText][SUCCESS ]:       /--or-* [Succ] TkInter is found
[TexText][INFO    ]: "textext/default_packages.tex" is not found in distribution, keep old file
[TexText][INFO    ]: Removing "C:\Users\username\AppData\Roaming\inkscape\extensions\textext"
[TexText][INFO    ]: Creating directory "C:\Users\username\AppData\Roaming\inkscape\extensions\textext"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\asktext.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\asktext.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\base.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\base.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\default_packages.tex" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\default_packages.tex"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\errors.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\errors.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\HarfBuzz-0.0.typelib" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\HarfBuzz-0.0.typelib"
[TexText][INFO    ]: Creating directory "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-bottom-center.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-bottom-center.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-bottom-left.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-bottom-left.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-bottom-right.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-bottom-right.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-middle-center.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-middle-center.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-middle-left.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-middle-left.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-middle-right.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-middle-right.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-top-center.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-top-center.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-top-left.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-top-left.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\alignment-top-right.svg.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\alignment-top-right.svg.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\logo-128x128.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\logo-128x128.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\logo-16x16.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\logo-16x16.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\logo-32x32.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\logo-32x32.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\icons\logo-64x64.png" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\icons\logo-64x64.png"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\LICENSE.txt" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\LICENSE.txt"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\README.txt" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\README.txt"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\requirements_check.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\requirements_check.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\texoutparse.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\texoutparse.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\textext.inx" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\textext.inx"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\utility.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\utility.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\VERSION" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\VERSION"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\win_app_paths.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\win_app_paths.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\__init__.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\__init__.py"
[TexText][INFO    ]: Copying "C:\Users\username\Downloads\inkscape\TexText-Windows-1.7.1\textext-1.7.1\textext\__main__.py" to "C:\Users\username\AppData\Roaming\inkscape\extensions\textext\__main__.py"
[TexText][SUCCESS ]: --> TexText has been SUCCESSFULLY installed on your system <--

**Comment by jcwinkler** [comment]
Thanks for the report!

> However, whenever I start textext Inkscape freezes than crashes and I can't understand why. The only thing I can think of is that there are some issues with the installation done with admin rights and the installation of textext done in the user folder.

I use TexText in the same configuration on several machines and never had problems. I do not think that this is the reason.

What puzzles me is the the following log entry:

```
[TexText][WARNING ]: Bad "inkscape" executable: "C:\Users\username\inkscape\inkscape-1.1.1_2021-09-20_3bf5ae0d25-x64\idir\bin\inkscape.exe"
```

Do you call the installer using  the `--inkscape-executable` option set to `C:\Users\username\inkscape\inkscape-1.1.1_2021-09-20_3bf5ae0d25-x64\idir\bin\inkscape.exe` ? Obviously it does not find Inkscape at the specified location and uses a fallback installation in the Program Files directory. Do you have multiple Inkscape installations on your machine?

**Comment by mattfack** [comment]
I made it work, here's how:

1. uninstall inkscape
2. remove the "inkscape" folder under C:\Program Files
3. remove the "inkscape" folder under C:\Users\username\AppData\Roaming
4. clean install of inkscape and textext

However, at first, I got the same log entry you noticed and I had no inkscape installed whatsoever. I had it installed with the 7zip folder, but I removed it completely.

PS: if you are not able to delete the folders because they are still used by some process, I strongly suggest you to download and install [process explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer)
